MAINTAINERS.md

```diff
@@ -1,6 +1,6 @@
-# Cloud Controller Maintainers
+# Nephe Maintainers
 
-This is the current list of maintainers for the `Cloud Controller` project. The maintainer
+This is the current list of maintainers for the Nephe project. The maintainer
```

Comment:
why add project with nephe? I saw at few other places

Reply:
I saw it in some theia docs, don't recall where exactly. I just checked, most of the places they refer to as Theia.
so we can go with it.

Also I have one question related to the image name. 
Should we say antrea/nephe or nephe image?   

I have used antrea/nephe since the image name is that way. And the commands to load image use antrea/nephe.